optuna/storages/rdb/storage.py

```diff
@@ -37,19 +37,23 @@
 class RDBStorage(BaseStorage):
     """Storage class for RDB backend.
 
-    This class is not supposed to be directly accessed by library users.
+    Note that library users can instantiate this class, but the fields and methods
```

Comment:
A detail but just "attributes" might be a more familiar term to Python users.
```suggestion
    Note that library users can instantiate this class, but the attributes
```
E.g.
```
In [1]: dir?
Docstring:
dir([object]) -> list of strings

If called without an argument, return the names in the current scope.
Else, return an alphabetized list of names comprising (some of) the attributes
of the given object, and of attributes reachable from it.
If the object supplies a method named __dir__, it will be used; otherwise
the default dir() logic is used and returns:
  for a module object: the module's attributes.
  for a class object:  its attributes, and recursively the attributes
    of its bases.
  for any other object: its attributes, its class's attributes, and
    recursively the attributes of its class's base classes.
Type:      builtin_function_or_method
```

Reply:
I didn't know the nuance. Thank you for letting me know it!